ecosystem/typescript/sdk/examples/typescript/bcs_transaction.ts

```diff
@@ -0,0 +1,87 @@
+/* eslint-disable no-console */
+import { AptosClient, AptosAccount, FaucetClient, BCS, TxnBuilderTypes } from 'aptos';
+
+const NODE_URL = process.env.APTOS_NODE_URL || 'https://fullnode.devnet.aptoslabs.com';
+const FAUCET_URL = process.env.APTOS_FAUCET_URL || 'https://faucet.devnet.aptoslabs.com';
+
+const {
+  AccountAddress,
+  TypeTagStruct,
+  ScriptFunction,
+  StructTag,
+  TransactionPayloadScriptFunction,
+  RawTransaction,
+  ChainId,
+} = TxnBuilderTypes;
+
+/**
+ * This code example demonstrates the process of moving test coins from one account to another.
+ */
+(async () => {
+  const client = new AptosClient(NODE_URL);
+  const faucetClient = new FaucetClient(NODE_URL, FAUCET_URL, null);
+
+  // Generates key pair for a new account
+  const account1 = new AptosAccount();
+  // Creates the account on Aptos chain and fund the account with 5000 TestCoin
+  await faucetClient.fundAccount(account1.address(), 5000);
+  let resources = await client.getAccountResources(account1.address());
+  let accountResource = resources.find((r) => r.type === '0x1::Coin::CoinStore<0x1::TestCoin::TestCoin>');
+  console.log(`account2 coins: ${(accountResource.data as any).coin.value}. Should be 5000!`);
+
+  const account2 = new AptosAccount();
+  // Creates the second account and fund the account with 0 TestCoin
+  await faucetClient.fundAccount(account2.address(), 0);
+  resources = await client.getAccountResources(account2.address());
+  accountResource = resources.find((r) => r.type === '0x1::Coin::CoinStore<0x1::TestCoin::TestCoin>');
+  console.log(`account2 coins: ${(accountResource.data as any).coin.value}. Should be 0!`);
```

Comment:
we should use some sort of assert here so that we can actually make these tests too.